uefi/src/allocator.rs

```diff
@@ -169,3 +169,17 @@ unsafe impl GlobalAlloc for Allocator {
         }
     }
 }
+
+#[cfg(feature = "unstable")]
```

Comment:
Since the implementation relies on `GlobalAlloc`, should this be gated on both `unstable` and the `global_allocator` feature?

Reply:
No, this isn't gated by the `global_allocator` feature. The `global_allocator` feature only gates the `#[global_allocator]` static